pkg/controller/dns/util.go

```diff
@@ -1,13 +1,20 @@
 package dns
 
 import (
+	"context"
 	"strings"
 
 	"github.com/Azure/aks-app-routing-operator/api/v1alpha1"
+	"github.com/Azure/aks-app-routing-operator/pkg/config"
 	"github.com/Azure/aks-app-routing-operator/pkg/manifests"
+	"github.com/Azure/aks-app-routing-operator/pkg/util"
+	"github.com/hashicorp/go-multierror"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 type ExternalDNSCRDConfiguration interface {
+	client.Object
```

Comment:
not a huge fan of embedding this way. better to choose exactly what we need usually

Reply:
this was just here for testing so worked around and removed it entirely!